scripts/download_image.py

```diff
@@ -285,6 +265,39 @@ def scan_file_for_download_list(urls: list, output: str, force: bool):
             report(next(count), total, future.result())
 
 
+def show_game_download_warning(workers: int,
+                               input_fn: callable = input) -> bool:
+    """
+    Shows warning in case user selected game as a source
+    :param input_fn: Input function for test or file reading
+    :param workers: Number of workers
+    :return:
+    >>> show_game_download_warning(10, input_fn=lambda x: "y")
+    False
+    >>> show_game_download_warning(10, input_fn=lambda x: "n")
+    True
+    """
+    choice = input_fn(
+            'WARNING: You specified {} workers which is over limit {'
+            '} to '
+            'download images from game website. This may lead you to '
+            'get '
+            'banned. If you accept it enter Y or enter N to exit '
+            'script. '
+            'If not please define number of threads with -w '
+            'parameter. '
+            'If you want to skip this warning you can enter -y while '
+            'calling script.:'.format(workers,
+                                      GAME_WORKER_LIMIT)).lower()
```

Comment:
'get ', '} to ' is too small chunk.
If line length was the limit, this text can move to global (top-level)
it's not became state, Python's string is an immutable object.

Reply:
Fixed by moving string into variable.

Reply:
```
    warning = 'WARNING: You specified {} workers which is over limit {} to ' \
              'download images from game website This may lead you to get ' \
              'banned. If you accept it enter Y or enter N to exit script. ' \
              'If not please define number of threads with -w parameter. If ' \
              'you want to skip this warning you can enter -y while calling ' \
              'script : '.format(workers, GAME_WORKER_LIMIT)
    warning = '\n'.join(warning[i:i + 92] for i in range(0, len(warning), 92))
```